gelf/gelf.go

```diff
@@ -0,0 +1,74 @@
+package gelf
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/aphistic/golf"
+	"github.com/yext/glog"
+	"github.com/yext/glog-contrib/raven"
+	"github.com/youtube/vitess/go/ratelimiter"
+)
+
+// Capture captures all events for a specific job and sends them to the gelf server.
+// Events sent at a higher rate than maxEventsPerSecond will be ignored.
+// The uri must have a udp or tcp scheme.
+func Capture(job, serverUri string, maxEventsPerSecond int, eventCh <-chan glog.Event) {
+	c, _ := golf.NewClient()
+	defer c.Close()
+
+	if c.Dial(serverUri) != nil {
+		panic("can't connect to gelf server")
+	}
+	logger, err := c.NewLogger()
+	if err != nil {
+		panic("can't initialize logger")
+	}
+
+	logger.SetAttr("job_name", job)
```

Comment:
Seems like this is pretty specific to our use case; maybe Capture could accept a map of key/value pairs to pass to logger.SetAttr ?